src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(health = 100, name) {
+    this.health = health;
```

Comment:
Health is equal to 100 by default, so you can write it in the constructor and don't pass it as an argument.